Composer/packages/client/src/components/AddRemoteSkillModal/BrowserModal.tsx

```diff
@@ -36,7 +37,14 @@ export const BrowserModal = (props) => {
         text={formatMessage('Browse')}
         onClick={onClickOpen}
       />
-      <input ref={inputFileRef} accept=".zip,.rar,.7zip" style={{ display: 'none' }} type="file" onChange={onChange} />
+      <input
+        ref={inputFileRef}
+        accept=".zip,.rar,.7zip"
```

Comment:
does JSZip support rar and 7zip?

Reply:
In addition, we do know that PVA is going to package it definitely as a zip. Is there a reason to support other formats?
@luhan2017

Reply:
Have checked, JSzip just support .zip, so I remove the rar & 7zip